ui/components-react/src/components-react/filter-builder/FilterBuilderRuleOperator.tsx

```diff
@@ -17,7 +17,7 @@ import {
 
 /**
  * Props for [[PropertyFilterBuilderRuleOperator]] component.
- * @internal
+ * @beta
```

Comment:
This module is not re-exported in a barrel, the API extractor doesn't pick it up.

Reply:
fixed